apps/admin/src/app/dashboard/reports/[id]/_components/comment.report.tsx

```diff
@@ -5,7 +5,6 @@ import type { ReportWithInfo } from '../_actions';
 
 export function CommentReport({ report }: { report: NonNullable<ReportWithInfo> }) {
   if (report.type !== 'COMMENT' || report.commentId === null || !report.comment) return null;
-  const rootId = report.comment.rootChallengeId ?? report.comment.rootSolutionId ?? -1;
```

Comment:
[Explanation] [`@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars`): This variable was unused.